### DISCUSSION TITLE
Emgu CV with VB.NET: The basics

### DISCUSSION THREAD

**Discussion by AH-Poh** [emgucv/emgucv#895] [General]
Hi guys.

Does it seem to me that Emgu is kinda unsuitable with VB.NET?

The tutorials on the Tubes are all in C#..

Can anyone here share a link on how to properly configure Emgu CV with VB.NET? The ones I found always offer deprecated code.

Thanks!

**Comment by FrankAVine** [comment]
Same problem here. Only two seriously old examples and they do not work.
